sf/draft procurement/stmt of qual.md

```diff
@@ -31,22 +31,26 @@ Play 2 – Address the Whole Experience from Start to Finish
   
 Play 3 – Make it Simple and Intuitive
   * The products or services for our customers/clients do not require knowledge of the organizational structure or our agency to complete the process.
-  * We coordinated with our customers/clients to make design and content decisions based on user research, usability testing, and analytics.
-  * For all user-facing features built over the last quarter, we coordinated with our customers/clients to use plain language that is familiar to the user and easy to understand.
+  * We coordinated with our customers/clients to make design and content decisions based on accessibility needs, user research, usability testing, and analytics.
+  * For all user-facing features built recently, we coordinated with our customers/clients to use plain language that is familiar to the user, easy to understand, and possible to translate across cultures and languages.
   
 Play 4 – Build the Service Using Agile and Iterative Practices
-  * For at least ½ of the projects/products for our customers/clients we launched a functioning “minimum viable product” (MVP) that took no longer than nine(?) months from the beginning of the project, using a “beta” or “test” period if needed.
+  * For at least half of the projects/products for our customers/clients we launched a functioning “minimum viable product” (MVP), using a “beta” or “test” period if needed.
   * We ensure the individuals building the service (product managers, developers, designers) communicate closely with each other throughout the process using techniques such as daily standups, launch meetings, and team chat tools.
-  * We release features and improvements multiple times each month for at least ½ of the projects/products for our customers/clients.
+  * We release features and improvements multiple times each month for at least half of the projects/products for our customers/clients.
   * We maintain a prioritized list of features and bugs, also known as a “backlog,” in an issue tracker that is accessible by everyone on the project.
   * We use a source code version control system.
   * We use code reviews to ensure quality for each release.
 
 Play 7 – Bring in Experienced Teams
-  * Our company’s teams include product managers who have built and launched user-centered, high-traffic digital services in previous projects.
-  * Our company’s teams include user experience designer(s) who have conducted one-on-one sessions with end users to understand user needs and translated those needs into a design for the digital service in previous projects.
-  * Our company currently has content strategist(s) who have developed user-centered, plain-language content in previous projects.
-  * Our company includes software engineer(s) who have used modern development and operations (DevOps) techniques like continuous integration and continuous deployment.
+
```

Comment:
- Modified this section to make it specific to the needs of a modern museum such as NMAAHC
- Corrected the entry for "content strategist" as what it identified was a copywriter
- Added entry for copywriter
- Added entry for art director, specifically calling out the need for expertise in branding, interaction design, movement, and animation.